.firebase/hosting.ZGlzdFxidWRnZXQtdHJhY2tlcg.cache

```diff
@@ -1,22 +1,30 @@
+dashboard-dashboard-module-es2015.js.map,1582406841833,55b0cd983c6e02739ffbeac2adca3426ab47a1ce4bddc242377b3c597dd0f1ca
+dashboard-dashboard-module-es5.js,1582406841825,8e883894f81c11b5dfdebada05a5ac7acd1ada0f953b619f0ed4c9da4f287c1f
+authentication-authentication-module-es2015.js,1582407800639,183112b067321e7e692425b7a9ccc6a869cc4f7030591567d5b4f051523411b5
+favicon.ico,1579442096896,2c19690e9587bae12f419b34d2edeecc76808099540a9c9f4ea6194116cfc8f7
+index.html,1582407800692,7ec6b91b4a4e06f36dd01d8f6be00b6bd0569db5bb60a27c53ca1383e189ba28
+dashboard-dashboard-module-es5.js.map,1582406841801,f261e7d22febd63e5dc137875f2dbedda69347e4fe763cc843fff408cd6a3189
+authentication-authentication-module-es5.js.map,1582407800625,c4372e58372f6ed0997eb3f77051d7bfcb679968585226e06821354c3f2e85f2
+main-es2015.js.map,1582406842113,7876320fb75f97904018f130a6af8750b8791ca097f7f295e1e6409a3ebb50cd
+main-es2015.js,1582406842118,069bd45b75237341ab2e8ed050c4003e847e6307d60c9092e7b5d603b5ad5e7d
+main-es5.js.map,1582406842098,4940fe74568fa30258984aa50165ab0362e933810238f460e84ce36cbaccc9cf
+main-es5.js,1582406842108,7fb2cd1fada6a7569b5a7c882cfb4e4a33ae29175628a2badca72d750a87ee68
+runtime-es2015.js,1582406840547,4d387bbdbd7c9ab9a750fa999bde972e4961eaab419f3cee5a68ca2a84a51f50
+runtime-es2015.js.map,1582406840537,2ddff3b46d2507a68c56ee15c5f4a77125dbea4138bd8c20aa6e7808336c9078
+runtime-es5.js,1582406840560,c028fa94c775568f25fc80a0c176a2bafa7642c7c3b9a5b2958eec7968c40b2e
+runtime-es5.js.map,1582406840553,d24f6add6be84f6b3d60286d86bee94a27c701425ab8bc053583638541e0e238
+styles-es2015.js,1582406841592,bee97f2ed8488d07339d1f5eb4d62b3607795c3f7bed3e6243f72611a9c8a226
+styles-es2015.js.map,1582406841585,1c4738fce5e59943fb0c0372672ce9852e09b226778d1adcb7b393b04cf701d8
+styles-es5.js,1582406841578,4ba3bd2d2052789677e7eb94938a724e74349f2dcf13c2fb4da101dfe65bd61f
+styles-es5.js.map,1582406841560,9d1fea7dddc0be07abaf85fd8b2b4e17a2a8f94601440795c05340357cb716d8
+authentication-authentication-module-es5.js,1582407800630,8aa5659ab2f83d31cb1846f05c70ed55aa48f3e10f597754a18c14ad5daf1e7f
+polyfills-es2015.js,1582406840469,bd34044b48a1178b6244f3d396bc5c4c3d31b78d5c6f1b34c3056256b71331e9
+polyfills-es2015.js.map,1582406840458,df35114a4297193a179ba94c31af00be4b502c3a770c0297c3f8405a7031ec09
+polyfills-es5.js,1582406843779,e9b80f8eeabd1d4a0310240595c8cabb0a97f3d33feabce30a4abb0d57390e3f
+polyfills-es5.js.map,1582406843762,f069e1d37fc78026d6eb246b0c648b9b7929b579d395de673a8cf963510b4879
+vendor-es2015.js,1582406854838,fb678162b1fe85ee392dc26f3f112cf2161ef39e570704fc96107c1303f2d98c
+vendor-es2015.js.map,1582406854768,1455a2191b1d7c6605f22d8d42e0474f21f25b4d214570b416c2664cd0e5c846
+vendor-es5.js,1582406854653,49320bd680c1e9c635b252f54afe7bf6e10bb7ca5d1299f8904ff9cf101383ab
+vendor-es5.js.map,1582406854560,0460294e763d6b742a8b63590c37a75049de101e5d5f7f73f020fb065640d370
```

Comment:
Do no include firebase cache in git repository. File is autogenerated and should not be committed. Please add whole .firebase directory into .gitignore.